docs/gettingStarted/setupGenericScheme.md

```diff
@@ -0,0 +1,217 @@
+# Setup Generic Scheme for a DAO
+
+## When to use GenericScheme or UGenericScheme
```

Comment:
This is not for now, but I think it could be useful to have a section of the docs that explains the difference between Universal schemes and non-universal schemes. and then just reference it here...

It could talk about the design principles in it etc...

Reply:
On it! Should be up for review soon 😉